src/Impostor.Server/Net/Inner/Objects/Systems/ShipStatus/SecurityCameraSystemType.cs

```diff
@@ -1,10 +1,17 @@
-﻿using Impostor.Api.Net.Messages;
+﻿using System.Collections.Generic;
+using Impostor.Server.Net.State;
+using Impostor.Api.Net.Messages;
 
 namespace Impostor.Server.Net.Inner.Objects.Systems.ShipStatus
 {
-    public class SecurityCameraSystemType : ISystemType
+    internal class SecurityCameraSystemType : ISystemType
     {
-        public byte InUse { get; internal set; }
+        private readonly Game _game;
+        public (HashSet<byte> UsingNow, HashSet<byte> StartedWatching, HashSet<byte> StoppedWatching) Players { get; internal set; } = (new (), new (), new ());
```

Comment:
Very messy.

Reply:
what's wrong?

Reply:
Split it up into three different properties, don't use a tuple for this.

Reply:
done https://github.com/Impostor/Impostor/pull/197/commits/0e49b389c01a95e9204de3c11ff430cf958c6886